docs/suppressing-instrumentation.md

```diff
@@ -6,99 +6,96 @@ You can disable the agent using `-Dotel.javaagent.enabled=false`
 ## Suppressing specific agent instrumentation
 
 You can suppress agent instrumentation of specific libraries by using
-`-Dotel.instrumentation.[name].enabled=false` where `name` is the instrumentation `name`
-(main or additional):
+`-Dotel.instrumentation.[name].enabled=false` where `name` is the corresponding instrumentation `name`:
 
-| Instrumentation name |
-|---------------------------|
-| akka-actor|
-| akka-http|
-| apache-camel|
-| apache-dubbo|
-| apache-httpasyncclient|
-| apache-httpclient|
-| armeria|
-| async-http-client|
-| aws-lambda|
-| aws-sdk|
-| axis2|
-| cassandra|
-| couchbase|
-| cxf|
-| dropwizard-views |
-| eclipse-osgi |
-| elasticsearch-rest|
-| elasticsearch-transport|
-| executor |
-| external-annotations |
-| finatra|
-| geode|
-| google-http-client|
-| grails|
-| grizzly|
-| grpc|
-| guava|
-| gwt|
-| hibernate|
-| http-url-connection |
-| httpclient |
-| hystrix|
-| jaxrs-client|
-| jaxrs|
-| jaxws|
-| jdbc |
-| jdbc-datasource |
-| jedis|
-| jetty|
-| jms|
-| jsp |
-| kafka |
-| khttp |
-| kotlinx-coroutines |
-| kubernetes-client|
-| lettuce|
-| liberty |
-| log4j|
-| logback|
-| methods |
-| metro|
-| mojarra|
-| mongo |
-| myfaces|
-| netty|
-| okhttp|
-| opentelemetry-annotations |
-| opentelemetry-api |
-| opentelemetry-metrics-api |
-| oshi |
-| play-ws|
-| play|
-| rabbitmq|
-| ratpack|
-| reactor-netty|
-| reactor|
-| rediscala|
-| rmi|
-| rocketmq-client|
-| rxjava2 |
-| scala-executors |
-| servlet|
-| spark|
-| spring-core|
-| spring-data|
-| spring-scheduling|
-| spring-webflux|
-| spring-webmvc|
-| spring-ws|
-| spymemcached|
-| struts|
-| tapestry|
-| tomcat|
-| twilio|
-| undertow|
-| vaadin|
-| vertx |
-| wicket|
+| Library/Framework | Instrumentation name |
+|-------------------|----------------------|
+| Akka Actor | akka-actor|
+| Akka HTTP | akka-http|
+| Apache Camel | apache-camel|
+| Apache Dubbo | apache-dubbo|
+| Apache HttpAsyncClient | apache-httpasyncclient|
+| Apache HttpClient | apache-httpclient|
+| Armeria | armeria|
+| AsyncHttpClient (AHC) | async-http-client|
+| AWS Lambda | aws-lambda|
+| AWS SDK | aws-sdk|
+| Apache Axis2 | axis2|
+| Apache Cassandra | cassandra|
+| Couchbase | couchbase|
+| Apache CXF | cxf|
+| Dropwizard Views | dropwizard-views |
+| Eclipse OSGi | eclipse-osgi |
+| Elasticsearch REST client | elasticsearch-rest|
+| Elasticsearch client | elasticsearch-transport|
+| java.util.concurrent | executor |
+| Additional tracing annotations | external-annotations |
+| Twitter Finatra | finatra|
+| Apache Geode | geode|
+| Google HTTP client | google-http-client|
+| Grails | grails|
+| Java EE Grizzly | grizzly|
+| GRPC | grpc|
+| Google Guava | guava|
+| Google Web Toolkit | gwt|
+| Hibernate | hibernate|
+| Java HTTP URL connection | http-url-connection |
```

Comment:
Consider just using the class name:
```suggestion
| Java `HttpURLConnection` | http-url-connection |
```

Reply:
I find class name under "Library/Framework" confusing.

Reply:
hm, tricky, I find the class name provides helpful specificity